### PR TITLE
Make some things public to support react-native

### PR DESCRIPTION
**Pull request by stemyke**
There is a react-native-community plugin that utilizes this library to make blur effect in react-native projects. However the problem is that this lib only works properly when using child Views inside BlurView. What I want to achieve to create a custom BlurView implementation for react-native which does exactly the same as BlurView in here, but instead of extending FrameLayout it can extend from ReactViewGroup class to utilize fancy flexbox like positioning.

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -44,3 +45,19 @@ task androidSourcesJar(type: Jar) {
     classifier = 'sources'
     from android.sourceSets.main.java.srcDirs
 }
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                groupId = 'com.github.stemyke'
```
Comment (Dimezis) [inline library/build.gradle:53]: Why would this publishing be included in this repo?

**Comment by Dimezis** [comment]
👍 I'm fine with these changes except that maven publishing

File: library/build.gradle (hunk 1 above)
Comment (stemyke) [reply library/build.gradle]: @Dimezis I was just figuring how should I publish the package, so I can import it into the project I mentioned. This is actually my first time I am doing something with an Android lib.

**Comment by stemyke** [comment]
@Dimezis Now I removed the publishing part. Can you please merge it then, and release a new version?

**Comment by Dimezis** [comment]
I have a question though, what's the point of introducing another BlurView based on `ReactViewGroup`, if you can just put `ReactViewGroup` into the `BlurView` as a child and position everything inside as you like?

**Comment by stemyke** [comment]
@Dimezis Thats a good question about the mentioned point (positioning), but ReactViewGroup does other things very well, for example border radiuses with custom drawables, which we could not achieve if we wrap it in the BlurView unless we implement it again in our custom view manager. Another thing to mention is that react-native uses these view managers to inject children into other ViewGroups where we dont know anything about the parent View, so we could only inject the child view in JavaScript which would make the lib more complex.

File: jitpack.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+jdk:
+  - openjdk11
```
Comment (Dimezis) [inline jitpack.yml:2]: Is this needed?

Comment (stemyke) [reply jitpack.yml:2]: No its not, it was also for publishing. Can you remove it or should I do it?

Comment (Dimezis) [reply jitpack.yml:2]: Please remove it and I'll try to release everything today

Comment (stemyke) [reply jitpack.yml]: Thanks, I removed it now.

**Review by Dimezis — APPROVED** [review]

**Comment by Dimezis** [comment]
It should be accessible on Jitpack soon https://github.com/Dimezis/BlurView/releases/tag/version-2.0.4

**Comment by stemyke** [comment]
> It should be accessible on Jitpack soon https://github.com/Dimezis/BlurView/releases/tag/version-2.0.4

@Dimezis Thank you very much!

**Pull request merged**